# Introduction #

The diverse clinical spectrum of diseases manifested by mitochondrial dysfunction demands proper interpretation and analysis of mtDNA variants. **mit-o-matic** is a tool designed to provide clinically relevant information distributed in disparate resources for understanding and annotating mtDNA variations. The pipeline has two components: an **offline** component that provides automation for alignment of NGS reads to the modified rCRS; and an **online** version that provides a comprehensive genetic report of mitochondrial variants, which includes information about the corresponding mutation, gene, associated phenotype and population frequency. It also reports the estimated haplogroup, disease associated variations and heteroplasmic sites.

![http://genome.igib.res.in/mitomatic/mito_home.png](http://genome.igib.res.in/mitomatic/mito_home.png)

**Citation:** Velarikkal, S.K. _et. al._ mit-o-matic: a comprehensive computational pipeline for clinical evaluation of mitochondrial variations from next-generation sequencing datasets (submitted 2014).

**Funding and Support:** This resource has been funded by Council of Scientific and Industrial Research (CSIR) - India, through Grant No. BSC0122 - CARDIOMED.
The resource is hosted by the Institute of Genomics and Integrative Biology (IGIB) - CSIR, and maintained by the Dr. Vinod Scaria Lab @ RNABiology.

**Copyright Â© 2014 CSIR-Institute of Genomics and Integrative Biology**